fineract-provider/config/checkstyle/checkstyle.xml

```diff
@@ -108,16 +113,8 @@
       
         <module name="SuppressWarningsHolder" />
         <module name="OuterTypeFilename"/>
-        <module name="IllegalTokenText">
-            <property name="tokens" value="STRING_LITERAL, CHAR_LITERAL"/>
-            <property name="format" value="\\u00(08|09|0(a|A)|0(c|C)|0(d|D)|22|27|5(C|c))|\\(0(10|11|12|14|15|42|47)|134)"/>
-            <property name="message" value="Avoid using corresponding octal or Unicode escape."/>
-        </module>
-        <module name="AvoidEscapedUnicodeCharacters">
```

Comment:
I've noticed that you have removed instead of moved this module.. was that intentionally? Why?

Reply:
This happened because I tested both of them simultaneously, but later followed our rule of one checkstyle per PR. It is unintentional :)

Reply:
@thesmallstar OK, then rebase and fix this?